content/cli/orch_cli/blueprints.md

```diff
@@ -310,6 +310,53 @@ Blueprints:
 ...
 {{< /highlight >}}
 
+### summary
+
+#### Usage 
+`cfy blueprints summary <target field> [optional sub-field] [OPTIONS]`
```

Comment:
I think that target in 'target field' is a little confusing, because you think target is something also you need to write

Reply:
Hmm... better as target_field? Does 'optional sub-field' suffer from the same issue?

Reply:
Just field could be a fair choice, or summarized field maybe